tests/pytests/unit/modules/test_aptpkg.py

```diff
@@ -213,6 +215,106 @@ def configure_loader_modules():
     return {aptpkg: {"__grains__": {}}}
 
 
+@pytest.fixture
+def deb822_repo_content():
+    return """# TEST SOURCE Deb822
+Types: deb
+URIs: http://cz.archive.ubuntu.com/ubuntu/
```

Comment:
We should have a test that has multiple URIs

Reply:
Hmm, good point, I didn't noticed that URI could be multi-valued with deb822, as I haven't seen any examples of apt sources with multiple URIs. Besides of tests it requires some adjustments in the code of `aptpkg` module as we have to expand it to separate entries for backward compatibility also. Will fix it.